zetaclient/evm/evm_signer.go

```diff
@@ -139,7 +139,18 @@ func (signer *Signer) Sign(
 	height uint64,
 ) (*ethtypes.Transaction, []byte, []byte, error) {
 	log.Debug().Msgf("TSS SIGNER: %s", signer.tssSigner.Pubkey())
-	tx := ethtypes.NewTransaction(nonce, to, big.NewInt(0), gasLimit, gasPrice, data)
+
+	tx := ethtypes.NewTx(&ethtypes.DynamicFeeTx{
+		ChainID:   big.NewInt(signer.chain.ChainId),
+		Nonce:     nonce,
+		To:        &to,
+		Value:     big.NewInt(0),
+		Gas:       gasLimit,
+		GasTipCap: gasPrice,
```

Comment:
I forgot about this change that was necessary or this test. I will create a separate PR for this change so we can discuss on it

Reply:
Yes, though it's fine, we're not planning new chain support right now

Reply:
FYI: I reverted the changes since it changes the behavior of ZetaClient which is out of scope for the PR. I disabled the test for now and track it here: https://github.com/zeta-chain/node/issues/1980